ydb/docs/en/core/dev/example-app/_includes/example-nodejs.md

```diff
@@ -1,261 +1,353 @@
+
+Depending on the rowMode parameter, the data can be retrieved in javascript form or as YDB structures.
+
+{% list tabs %}
+
+- rowMode: RowType.Native
+
+  ```ts
+  async function selectNativeSimple(driver: Driver, logger: Logger): Promise<void> {
+      logger.info('Making a simple native select...');
+      const result = await driver.queryClient.do({
+          fn: async (session) => {
+              const {resultSets} =
+                  await session.execute({
+                      // rowMode: RowType.Native, // Result set cols and rows returned as native javascript values. It's default behaviour
+                      text: `
+                          SELECT series_id,
+                                 title,
+                                 release_date
+                          FROM ${SERIES_TABLE}
+                          WHERE series_id = 1;`,
+                  });
+              const {value: resultSet1} = await resultSets.next();
+              const rows: any[][] = []
+              for await (const row of resultSet1.rows) rows.push(row);
+              return {cols: resultSet1.columns, rows};
+          }
+      });
+      logger.info(`selectNativeSimple cols: ${JSON.stringify(result.cols, null, 2)}`);
+      logger.info(`selectNativeSimple rows: ${JSON.stringify(result.rows, null, 2)}`);
+  }
+  ```
+
+- rowMode: RowType.Ydb
+
+  ```ts
+  async function selectTypedSimple(driver: Driver, logger: Logger): Promise<void> {
+      logger.info('Making a simple typed select...');
+      const result = await driver.queryClient.do({
+          fn: async (session) => {
+              const {resultSets} =
+                  await session.execute({
+                      rowMode: RowType.Ydb, // enables typedRows() on result sets
+                      text: `
+                          SELECT series_id,
+                                 title,
+                                 release_date
+                          FROM ${SERIES_TABLE}
+                          WHERE series_id = 1;`,
+                  });
+              const {value: resultSet1} = await resultSets.next();
+              const rows: Series[] = [];
+              // Note: resultSet1.rows will iterate YDB IValue structures
+              for await (const row of resultSet1.typedRows(Series)) rows.push(row);
+              return {cols: resultSet1.columns, rows};
+          }
+      });
+      logger.info(`selectTypedSimple cols: ${JSON.stringify(result.cols, null, 2)}`);
+      logger.info(`selectTypedSimple rows: ${JSON.stringify(result.rows, null, 2)}`);
+  }
+  ```
+
+{% endlist %}
 
 {% include [param_queries.md](steps/06_param_queries.md) %}
 
-Here's a code sample that shows how to use the `Session.executeQuery()` method with the queries and parameters
-prepared by `Session.prepareQuery()`.
+There is no explicit Prepared Query option in the Query Service. YDB determines the necessity of using this mode by
+itself.
```

Comment:
```suggestion
```
We are talking about the query service in documentation only if it is absolutely necessary.